### ISSUE TITLE
Attribute "shape" already defined with incompatible format

### ISSUE THREAD

**Issue by adrielcafe** [sharish/ShimmerRecyclerView#5]
Hello!

I've tried to use your lib, but there is an error while building the project. Seems like another lib that I'm using has the same `shape` attribute, but of another type (enum).

```
Error:(906) Attribute "shape" already defined with incompatible format.
```

Can you use a prefix like `shimmer_shape` to avoid errors? It would be good if all attributes also had too.

Many thanks!

**Comment by adrielcafe** [comment]
Fixed on PR #6

**Comment by alouanemed** [comment]
I am getting the same error :/
